docs/API.md

```diff
@@ -0,0 +1,369 @@
+# Hydra Auction Frontend API
+
+This document describes the API of the frontend glue library 
+that allows the Hydra Auction frontend to execute auction-related operations (also known as "contracts"). 
+The library is based on CTL. 
+
+## Executing operations
+
+The library can be used by importing a set of types and functions from `hydra-auction-offchain` package 
+as shown on a test page in the `/demo` folder.
+
+Operations are exposed as asynchronous functions (eponymous to names in this document) 
+that yield a value of type `ContractOutput` (see below).
```

Comment:
That's not exactly true for all exposed operations, since some of them (such as `queryAuctions` or `discoverBidders`) do not define typed errors and therefore do not return `ContractOutput`.

```suggestion
and most of them yield a value of type `ContractOutput` (see below).
```